### PR TITLE
Ability to Change Story Name

### PR DESCRIPTION
**Pull request by mattvv**
This lets you edit the story name in the card view

### CODE REVIEW

File: components/StoryCard.tsx [hunk 1]
```diff
@@ -28,6 +31,13 @@ const StyledSpan = styled.span`
   color: #3291ff;
 `;
 
+const Title = styled.textarea`
```
Comment (mattvv) [inline components/StoryCard.tsx:34]: @shc023 @RobertoValle31  @jansen0825 - do you know a way I can have the textarea resize as the content gets bigger? it seems to be fixed

File: components/StoryCard.tsx [hunk 2]
```diff
@@ -56,7 +81,12 @@ const StoryCard = ({ story, state, index, addFilter }: StoryCardParams): JSX.Ele
               width="250px"
               hoverable
               style={{ borderColor: borderColors[story.story_type] || 'gray' }}
-              onClick={() => dispatch(selectStory(story))}>
+              onClick={e => {
+                if (e.target.nodeName === 'TEXTAREA') {
+                  return;
+                }
```
Comment (shc023) [inline components/StoryCard.tsx:87]: (not for this PR) we'll want a more reusable method to block modal opening on clicking on owner/tag filters too.

File: components/StoryCard.tsx (hunk 1 above)
Comment (shc023) [reply components/StoryCard.tsx:34]: native Textarea you'll have to use JS for that. Geist does have a `TextArea` component but it looks like that also doesn't auto-expand..

**Review by shc023 — APPROVED** [review]

File: components/StoryCard.tsx (hunk 1 above)
Comment (jansen0825) [reply components/StoryCard.tsx:34]: @shc023 makes sense. Did you try forcing it to a 100% width?

**Review by jansen0825 — APPROVED** [review]
👍

File: components/StoryCard.tsx (hunk 1 above)
Comment (mattvv) [reply components/StoryCard.tsx:34]: Ok, I'll leave that to you @shc023 in a future pr, will merge for now

Comment (RobertoValle31) [reply components/StoryCard.tsx:34]: @mattvv  Maybe you can do this. Use a div and add the 'role' prop. You will need to add some border to the div in order to make it visible. You can find more info [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/textbox_role)
Screen Shot 2020-10-15 at 2 02 32 PM

**Pull request merged**